assets/css/site/header.css

```diff
@@ -71,7 +71,7 @@
     margin-left: 3rem;
 }
 
-.site-header .menu-item:nth-child(n+6) {
+.site-header .menu-item:nth-child(n + 6) {
```

Comment:
No reformatting, please. I'd like to keep in sync with upstream template as much as possible.